packages/ui/src/components/va-sidebar/VaSidebar.vue

```diff
@@ -7,7 +7,7 @@
     @mouseenter="updateHoverState(true)"
     @mouseleave="updateHoverState(false)"
   >
-    <div class="va-sidebar__menu" :style="`width: ${computedWidth};`">
```

Comment:
Sidebar "100% width" story is broken (aslwo was before your pr).

<img width="424" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/44facbf1-4c11-47ff-a51b-f8e8ef6db2e6">